torch/testing/_internal/opinfo/definitions/signal.py

```diff
@@ -301,17 +360,68 @@ def make_signal_windows_opinfo(
             ),
             DecorateInfo(
                 unittest.expectedFailure,
-                "TestNNCOpInfo",
-                "test_nnc_correctness",
+                "TestSchemaCheckModeOpInfo",
+                "test_schema_correctness",
                 dtypes=[torch.float16],
                 device_type="cpu",
             ),
-            *skips,
         ),
-    )
-
-
-op_db: List[OpInfo] = [
+    ),
+    make_signal_windows_opinfo(
+        name="signal.windows.bartlett",
+        ref=reference_signal_window(scipy.signal.windows.bartlett)
+        if TEST_SCIPY
+        else None,
+        sample_inputs_func=sample_inputs_window,
+        reference_inputs_func=reference_inputs_window,
+        error_inputs_func=error_inputs_window,
+    ),
+    make_signal_windows_opinfo(
+        name="signal.windows.blackman",
+        ref=reference_signal_window(scipy.signal.windows.blackman)
+        if TEST_SCIPY
+        else None,
+        sample_inputs_func=sample_inputs_window,
+        reference_inputs_func=reference_inputs_window,
+        error_inputs_func=error_inputs_window,
+        skips=(
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestDecomp",
+                "test_comprehensive",
+                dtypes=[torch.float16],
+                device_type="cpu",
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestMeta",
+                "test_dispatch_meta",
+                dtypes=[torch.float16],
+                device_type="cpu",
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestMeta",
+                "test_meta",
+                dtypes=[torch.float16],
+                device_type="cpu",
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestMeta",
+                "test_dispatch_symbolic_meta",
+                dtypes=[torch.float16],
+                device_type="cpu",
+            ),
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestSchemaCheckModeOpInfo",
+                "test_schema_correctness",
+                dtypes=[torch.float16],
+                device_type="cpu",
```

Comment:
Arent' all thse already added in `make_signal_windows_opinfo`?

Reply:
Same for the other OpInfos here.

Reply:
@lezcano I moved those out of the `make_signal_windows_opinfo` definition. I'm not sure why the diff is not being shown 🤔 

Take a look at the `master` file:
https://github.com/pytorch/pytorch/blob/dc00bb51b8d370bf3891f0edb2c6e0c2914e329a/torch/testing/_internal/opinfo/definitions/signal.py#L209-L311

Reply:
my point is the opposite then. Why would you move them out, rather than have them all in `make_signal_windows_opinfo`?

Reply:
Hi @lezcano. Apologies for the late response. The reason why I moved them out of `make_signal_windows_opinfo` is that not all the windows need to skip those tests. See `bartlett` window. It's not very pretty but it is what it is. For example, most of the windows use `torch` functions that don't support `torch.half`.

Anything that you could suggest?

Reply:
Should we assert that the dtypes we support are only `float`, `double`, `cfloat` and `cdouble` for all the windows? That would surely save us from quite a bit of pain, and we can always generalise them if people ask for that.

Reply:
@lezcano do you mean adding an additional window check to each window? We could do that, I guess. How would we generalize them?